scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -20,18 +22,39 @@ case class ExplicitReturnTypes(mirror: Mirror)
 
   def defnName(defn: Defn): Option[Name] = Option(defn).collect {
     case Defn.Val(_, Seq(Pat.Var.Term(name)), _, _) => name
+    case Defn.Var(_, Seq(Pat.Var.Term(name)), _, _) => name
     case Defn.Def(_, name, _, _, _, _) => name
   }
 
+  def visibility(mods: Traversable[Mod]): MemberVisibility =
+    mods
+      .collect {
+        case _: Mod.Private => MemberVisibility.Private
+        case _: Mod.Protected => MemberVisibility.Protected
+      }
+      .headOption
+      .getOrElse(MemberVisibility.Public)
+
+  def kind(defn: Defn): Option[MemberKind] = Option(defn).collect {
+    case _: Defn.Val => MemberKind.Val
+    case _: Defn.Def => MemberKind.Def
+    case _: Defn.Var => MemberKind.Var
```

Comment:
I would think `Implicit` should not be an option and be always applied, since `Implicit` should always have explicit return type / type annotated even in Scala 2.x , like:
https://issues.scala-lang.org/browse/SI-9130

While I believe there is no use case that implicit should NOT have explicit return type, please let me know if there is.